ast/expressions.go

```diff
@@ -715,6 +715,13 @@ func (n *PositionExpr) Accept(v Visitor) (Node, bool) {
 	if skipChildren {
 		return v.Leave(newNode)
 	}
+	if n.N != nil {
+		node, ok := n.N.Accept(v)
+		if !ok {
+			return n, false
+		}
+		n.N = node.(ExprNode)
+	}
 	n = newNode.(*PositionExpr)
```

Comment:
Put this line before the added check?

Reply:
Thank you for the comment. I've updated the PR.

Reply:
I mean like this:
```go
n = newNode.(*PositionExpr)
if n.N != nil {
    node, ok := n.N.Accept(v)
    if !ok {
        return n, false
    }
    n.N = node.(ExprNode)
}
```
we should modify `newNode.N` instead of `n.N`?

Reply:
Thank you for the correction. I've updated the PR.